### PR TITLE
crash multiline header fix

### PR DESCRIPTION
**Pull request by Taliik**
fixed the problem with the formatting if the first row was empty, in line 585 & 586
also added some formatting tests

### CODE REVIEW

File: src/crate/crash/tabulate.py [hunk 1]
```diff
@@ -506,24 +547,61 @@ def _align_column(strings, alignment, minwidth=0, has_invisible=True):
...
+                "\n".join([padfn(maxwidth, s) for s in ms.splitlines()])
+                for ms in strings]
+        else:
+            # enable wide-character width corrections
+            s_lens = [max((len(s) for s in re.split("[\r\n]", ms))) for ms in strings]
+            visible_widths = [maxwidth - (w - l) for w, l in zip(s_widths, s_lens)]
+            # wcswidth and _visible_width don't count invisible characters;
+            # padfn doesn't need to apply another correction
+            if strings[0] == '':
+                strings[0] = ' '
```
Comment (mxm) [inline src/crate/crash/tabulate.py]: This seems a bit hacky. Can't we enable support for empty string padding in `padfn`?

Comment (mxm) [reply src/crate/crash/tabulate.py]: Are these the only changes from the old multiline formatting code?

Comment (Taliik) [reply src/crate/crash/tabulate.py]: Yes it kinda is. I don't think padfn supports empty strings, but i will have another look to see if i can find a better solution.
yes these are the only changes.

File: CHANGES.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ Changes for crash
 Unreleased
 ==========
 
+ - added back support for multiline table headers
```
Comment (mxm) [inline CHANGES.txt]: You could capitalize the first letter here.

**Pull request merged**